### PR TITLE
update integration tests

### PR DESCRIPTION
**Pull request by CatalinFaurPaul**
## Type
- [x] Integration test

## Problem setting
- Increase integration tests 

  
## Proposed Changes
- Update integration tests

### CODE REVIEW

File: src/test/integration/accounts.e2e-spec.ts [hunk 1]
```diff
@@ -55,6 +52,10 @@ describe('Account Service', () => {
       const accountsCount = await accountService.getAccountsCount();
       expect(typeof accountsCount).toBe('number');
     });
+
+    it(`should return a number`, async () => {
```
Comment (tanghel) [inline src/test/integration/accounts.e2e-spec.ts:56]: this test does nothing

File: src/test/integration/api.config.e2e-spec.ts [hunk 2]
```diff
@@ -30,11 +32,15 @@ describe('API Config', () => {
     const moduleRef = await Test.createTestingModule({
       imports: [PublicAppModule],
     }).compile();
-
+    configService;
```
Comment (tanghel) [inline src/test/integration/api.config.e2e-spec.ts:35]: this line should be removed

File: src/test/integration/blocks.e2e-spec.ts [hunk 3]
```diff
@@ -25,11 +25,11 @@ describe('Blocks Service', () => {
 
   describe('Blocks', () => {
     it('blocks should have hash, epoch and shard', () => {
-      for (const block of blocks) {
+      blocks.forEach(block => {
```
Comment (tanghel) [inline src/test/integration/blocks.e2e-spec.ts:28]: please use for ... of instead of .forEach

Comment (CatalinFaurPaul) [reply src/test/integration/blocks.e2e-spec.ts]: fixed

File: src/test/integration/accounts.e2e-spec.ts (hunk 1 above)
Comment (CatalinFaurPaul) [reply src/test/integration/accounts.e2e-spec.ts]: fixed

**Pull request merged**